frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunDrawerRightContent.tsx

```diff
@@ -37,12 +37,7 @@ const PipelineRunDrawerRightContent: React.FC<PipelineRunDrawerRightContentProps
   return (
     // TODO: Revisit below approach, either to further to look into what caused the content to not render or
     // to restructure the code -- see https://issues.redhat.com/browse/RHOAIENG-23537
-    <div
-      className="pf-v6-c-drawer__panel"
-      data-testid="pipeline-run-drawer-right-content"
-      // TODO: look into removing this inline style; PF Drawers should handle height/scrolling by default
-      style={{ height: '100%', overflowY: 'auto' }}
-    >
+    <div className="pf-v6-c-drawer__panel-main" data-testid="pipeline-run-drawer-right-content">
```

Comment:
Wasn't this a PF component before? Are we stuck with the `div` + class name?

Reply:
Yes, this was previously a PatternFly component, and we're currently using the div + classname as a workaround. This PR focuses on removing the page style overrides, while the changes to this component are tracked separately here: https://issues.redhat.com/browse/RHOAIENG-23537

I’d be glad to incorporate it into this PR, but it will likely require some refactoring of the Drawer component, which is outside the current scope. We could also consider addressing it as a follow-up. What do you think, @andrewballantyne ?

Reply:
I think if we revert these two, it's better imo -- do that with the other issue

Reply:
That sounds like a good idea @andrewballantyne - I'm okay with closing this PR and moving issue 23580 over to 23537 as a sub-task. I’ll confer with the PF enablement team and anyone else interested in these discussions regarding the improvements to the Page implementation and the Drawer structure, and we can aim to include both revisions in one PR.

Reply:
That works for me -- the only real value add here I think is the CSS to explicit `null` call to the sidebar for Page... which I like, but don't like at the same time haha.

Delaying the whole aspect works for me if we can get better results out of PF.